docs/guides/workspace-and-project-management.rst

```diff
@@ -0,0 +1,39 @@
+================================
+Workspace and Project Management
+================================
+
+Compute Engine (CE) organises workflow runs into ``Workspaces`` and ``projects``.
+As such, in order to submit or interact with a workflows run from the CLI, it is necessary to specify the ``Workspace`` and ``Project`` that contains it [1]_.
```

Comment:
**suggestion**:
1. Typography (` ``foo`` ` vs `*foo*`)
2. CE requires specifying the workspace and project regardless of the way the user interacts with CE. It's true for both CLI and the Python API.
3. It would be nice to mention the reason why these concepts exist and how they relate to the meaning users might already have in their minds. My immediate association is a git repo for a "project" and an IDE window for a "workspace". Both are completely invalid when it comes to Orquestra projects and workspaces.

```suggestion
Compute Engine (CE) groups user-submitted entities like workflow runs, or secrets into *workspaces*.
The main reason for associating entities with workspaces is sharing data with other users.
You can invite other users to a workspace via Orquestra Portal to let them see your stuff.

Additionally, workflow runs on CE have another level of organisation: they're grouped by *projects*.
This categorisation is related to the Orquestra Studio's UI.

When you're using Orquestra Workflow SDK from Orquestra Studio, the workspace and project are set automatically.
When you're on a local machine and you're interacting with CE, you need to specify workspace and project you're interested in.

.. note::
   Orquestra workspaces and projects are purely server-side concepts.
   There's no built-in 1:1 correspondence with a git repo or your local IDE project.
```